test/test_generate/test-autograder/autograder-custom-env/requirements.txt

```diff
@@ -0,0 +1,18 @@
+datascience
```

Comment:
please add the path to this file (from the root of the repo) to the list `FILES_WITH_VERSIONS` in `release.py` (in the root of the repo, not in the package)